nwscraper/main.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+const cheerio = require("cheerio");
+const semver = require("semver");
+
+const scraper = require("./scraper");
+
+const RELEASE = /^v\d+\.\d+\.\d+\/$/;
+const MIN_VERSION = process.env.MIN_NW_VERSION || "0.13.0";
+
+scraper("http://dl.nwjs.io/")
+  .then(function(body) {
+    const $ = cheerio.load(body);
+
+    const versions = $("a")
+      .map(function(i, link) {
+        return $(link).text();
+      })
+      .get()
+      .filter(function(version) {
+        return RELEASE.test(version);
```

Comment:
A method reference could be more concise here, if you don't consider it to be a degradation of readability.